src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,12 +28,18 @@ public class RobotContainer {
 	// The robot's subsystems and commands are defined here...
 	private final Drivetrain drivetrain = new Drivetrain(driverController);
 
+	//declared NAVX
+	public final NAVX navx;
+
 	/**
 	 * The container for the robot. Contains subsystems, OI devices, and commands.
 	 */
 	public RobotContainer() {
 		// Configure the trigger bindings
 		configureBindings();
+
+		//initialized?
+		navx = new NAVX(null);
```

Comment:
see comments about Arguments in NavX Class - there should be nothing passed here. NavX can actually be initialized like the drivetrain class in line 29